### ISSUE TITLE
[MT5] - measure_operator_cost not implemented for op mul_165_1000792_2000792

### ISSUE THREAD

**Issue by goliaro** [flexflow/FlexFlow#341]
When running the MT5 model with the following command: `flexflow_python examples/python/pytorch/mt5/mt5_ff.py -ll:py 1 -ll:gpu 1 -ll:fsize 14000 -ll:zsize 4096 --simulator-workspace-size 7057475584` I get the `measure_operator_cost` not implemented error below:

```bash
[Measure Dropout] name(decoder_dropout_1_1000793_2000793) forward_time(0.0346) backward_time(0.0194)
measure_operator_cost not implemented for op mul_165_1000792_2000792 (type 79). Please report this issue to the FlexFlow developers.
Aborted (core dumped)
```

Full output:
<details>
<summary>Toggle here </summary>
<br>

root@050487f4cf2e:/usr/FlexFlow# flexflow_python examples/python/pytorch/mt5/mt5_ff.py -ll:py 1 -ll:gpu 1 -ll:fsize 14000 -ll:zsize 4096 --simulator-workspace-size 7057475584
[0 - 7f3bec743700]    0.398907 {3}{Mapper}: Enabled Control Replication Optimizations.
[0 - 7f3bec743700]    0.399016 {3}{Mapper}: Enabled Control Replication Optimizations.
[0 - 7f3bec743700]    0.399050 {3}{Mapper}: Enabled Control Replication Optimizations.
Using cffi flexflow bindings.
Using flexflow python
start top-level task
top-level task
workSpaceSize (1024 MB)
Downloading: 100%|█████████████████████████████████████████████████████████████████████████████████████| 1.12G/1.12G [00:21<00:00, 56.7MB/s]
Loading data...
Tracing the model...
/opt/conda/lib/python3.9/site-packages/torch/fx/symbolic_trace.py:434: UserWarning: Was not able to add assertion to guarantee correct inputs to specialized function. It is up to the user to make sure that your inputs match the inputs you specialized the function with.
  torch.warnings.warn(
input_ids; ; view,; INPUT
attention_mask; ; getitem,getitem_1,; INPUT
decoder_input_ids; ; view_33,; INPUT
view; input_ids,; shared,; VIEW; -1; 48
shared; view,; encoder_dropout,; EMBEDDING; 250112; 512
getitem; attention_mask,; to,; GETITEM; slice(None, None, None); None; None; slice(None, None, None)
to; getitem,; sub,; TO; torch.float32
sub; to,; mul,; SCALAR_SUB; 1.0
mul; sub,; add_1,; SCALAR_MULTIPLY; -10000.0
encoder_dropout; shared,; to_1,mul_1,add_3,; DROPOUT; 0.1
mul_2; encoder_dropout,; encoder_block_0_layer_0_self_attention_q,encoder_block_0_layer_0_self_attention_k,encoder_block_0_layer_0_self_attention_v,; LAYER_NORM
encoder_block_0_layer_0_self_attention_q; mul_2,; view_1,; LINEAR; 384; 10; 0
view_1; encoder_block_0_layer_0_self_attention_q,; transpose,; VIEW; 64; -1; 6; 64
transpose; view_1,; matmul,; TRANSPOSE; 1; 2

...
...
...

decoder_block_7_layer_2_dense_relu_dense_dropout; mul_163,; decoder_block_7_layer_2_dense_relu_dense_wo,; DROPOUT; 0.1
decoder_block_7_layer_2_dense_relu_dense_wo; decoder_block_7_layer_2_dense_relu_dense_dropout,; decoder_block_7_layer_2_dropout,; LINEAR; 512; 10; 0
decoder_block_7_layer_2_dropout; decoder_block_7_layer_2_dense_relu_dense_wo,; add_139,; DROPOUT; 0.1
add_139; add_135,decoder_block_7_layer_2_dropout,; to_43,mul_164,; ADD
mul_165; add_139,; decoder_dropout_1,; LAYER_NORM
decoder_dropout_1; mul_165,; lm_head,; DROPOUT; 0.1
lm_head; decoder_dropout_1,; output,; LINEAR; 250112; 10; 0
output; lm_head,; ; OUTPUT
Compiling the model...
num_nodes = 1 num_gpus_per_node = 1
[Measure Dropout] name(decoder_dropout_1_1000793_2000793) forward_time(0.0346) backward_time(0.0194)
measure_operator_cost not implemented for op mul_165_1000792_2000792 (type 79). Please report this issue to the FlexFlow developers.
Aborted (core dumped)

</details>

**Comment by lockshaw** [comment]
I believe this is caused by #346, once I've merged the fix for that let's try running this again and see if it still happens.

**Comment by lockshaw** [comment]
Try again after merging #622 and #680
